docs/src/routes/docs/[...3]modules/core.md

```diff
@@ -195,6 +195,14 @@ type ConnectModalOptions = {
    * Defaults to `https://ethereum.org/en/wallets/find-wallet/#main-content`
    */
   iDontHaveAWalletLink?: string
+  /**
+   * Define support for Unstoppable Domains resolutions
+   * after a user connects. Similar to ens, uns can be used for users who
+   * have minted an Unstoppable Domain and associated it with their wallet.
+   * ENS resolution takes precedent over UNS
+   * Defaults to true
```

Comment:
This should be `false`! @Adamj1232

Reply:
This is fixed on the release branch, #1557